dev/io.openliberty.jaxws.security_fat.ssl/publish/files/JaxWsTransportSecurityServer/serverConfigs/basicAuthWithSSL.xml

```diff
@@ -0,0 +1,58 @@
+<server>
```

Comment:
Same comment about the server.xml files, make sure this project is using whatever files are here, and delete the ones not in use in this project.